types/jest/jest-tests.ts

```diff
@@ -1120,11 +1123,11 @@ describe('', () => {
         // $ExpectError
         nonPromiseMatchers.toMatchInlineSnapshot({notthing: extendedExpect.any(Boolean)});
 
-        let promiseMatchers: jest.PromiseMatchers<typeof matchers> = matchers.rejects;
+        let promiseMatchers: jest.PromiseMatchers<any> = matchers.rejects;
```

Comment:
can you put back `typeof matchers` ( or remove the type annotation ) and the $ExpectType